src/bidiMapper/domains/events/SubscriptionManager.ts

```diff
@@ -130,7 +130,26 @@ export class SubscriptionManager {
 
     // Get all the subscription priorities.
     const priorities: number[] = relevantContexts
-      .map((c) => contextToEventMap.get(c)?.get(eventMethod))
+      .map((context) => {
+        // Get the priority for exact event name
+        const priority = contextToEventMap.get(context)?.get(eventMethod);
+        // For CDP we can't provide specific event name when subscribing
+        // to the module directly.
+        // Because of that we need to see event `cdp` exits in the map.
+        if (eventMethod.startsWith(ChromiumBidi.BiDiModule.Cdp)) {
```

Comment:
We use `name.startsWith(ChromiumBidi.BiDiModule.Cdp)` in at least two different places (plus see the comment in the other place about the dot).

Should we extract this into an `isCdpEventName` helper?

Reply:
I think with your suggestion for using a better way to determine if its under the `cdp` module scope yes, but for a single like I would be opposed to it.